README.md

```diff
@@ -41,6 +41,14 @@ sudo apt update
 sudo apt install red-cli
 ```
 
+**ArchLinux**
+
+Install from the [redmine-cli-go AUR package](https://aur.archlinux.org/packages/redmine-cli-go)
```

Comment:
Since there were already some redmine CLI tools in the AUR (https://aur.archlinux.org/packages?O=0&K=redmine) I decided to name it `redmine-cli-go`. Are there better names? Maybe just `red-cli`? But then the repository's name wouldn't match.